Dockerfile.dev

```diff
@@ -0,0 +1,9 @@
+FROM node:latest
```

Comment:
This image is beefy 900mb+ but node-sass is needy, and I'm too lazy to track down all the dependencies we'd need to add to alpine 😅 

But maybe better to at least pin it to node 10 (or 11 if that's what you're gunnin for) 👍

Reply:
Good idea.  I'll pin to `11` for now and set up a new issue to see if we can reduce the image size some more: https://github.com/SharpNotions/pantry-list-frontend/issues/48